### ISSUE TITLE
Can you support compiling iPad applications?

### ISSUE THREAD

**Issue by hileel** [flowtyone/flowty-realtime-lcm-canvas#5]
Can you support compiling iPad applications?

**Comment by Jimoh10** [comment]
JIMOH OWOLABI AHMED
                     TEL:09019467355
9. AMUDAT ABEKE STREET BARIGA, LAGOS STATE.

              APPLICATION LETTER

GOOD-DAY MANAGEMENT
         I hereby apply for the vacacant position in this company I would like to help the company archive it's mission of making a healthy production to reach it's goal.
 
I've worked in other companies in my career, and we reach the opportunity to wear many art and work with the team to success. In my latest role as an administrative assistant at zealous building company, I saved my employer of temp worker with good resource, implementing a self scheduling system for customers/rep that cut-down on swift-shift plus I'm ready to learn perfect my skill with no negatives. Therefore, I really hope there is a chance for interview for us to explain how the potential and working category and I promise to be of well standard to bring us good out come. Thank you 

                                            Yours sincerely
                       Jimoh Owolabi Ahmed Adisa
